### PR TITLE
Reformat table to be able to fit more people #13

### PR DESCRIPTION
**Pull request by kentcdodds**
cc @ben-eb, @Jameskmonger, @jfmengels, @chrissimpkins, @bogas04 

Would love to get your feedback on this!

### CODE REVIEW

**Comment by kentcdodds** [comment]
See [this page](https://github.com/kentcdodds/all-contributors/blob/pr/reformat-table/README.md) to preview what this looks like.

I realize that it's quite a bit of text and hard to format by hand, but I'm hoping that @jfmengels's tool can do the hard formatting stuff for us :+1:

**Comment by kentcdodds** [comment]
cc @krmaxwell, would you like to give this a look as well?

**Comment by ben-eb** [comment]
Much nicer. :+1:

**Comment by chrissimpkins** [comment]
Much better!  Great idea. That layout will allow use in projects with a significantly greater number of contributors than the previous format.

**Comment by kentcdodds** [comment]
> That layout will allow use in projects with a significantly greater number of contributors than the previous format.

:boom:

**Comment by bogas04** [comment]
It looks nice and classy! 
However I think keeping the links away from the profile of user isn't friendly to modifications. For example moving one profile from one location to another, or just editing the roles would require adjusting code at 2 places or manually counting and repeated previews. Hence I think all profile data and emoji should be kept in one cell.

Further, I guess the contributors list can exceed 12-14 too in certain cases, so we should instead provide different templates that fits different scenarios, else we'll have to make another PR for large team sizes. One fit doesn't fit all after all

For 1-4: 

Contributor | Contributions
:---: | :---:
[Kent C. Dodds<br />Kent C. Dodds](http://kentcdodds.com) | [📖](https://github.com/kentcdodds/all-contributors/commits?author=kentcdodds) 👀 ❓
[James Monger<br />James Monger](http://github.com/Jameskmonger) | [📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger)
[Ben Briggs<br />Ben Briggs](http://beneb.info) | [📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb)

For 5-6: 

| [Kent C. Dodds<br /><sub>Kent C. Dodds</sub>](http://kentcdodds.com) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=kentcdodds) 👀 ❓ | [Divjot Singh<br /><sub>Divjot Singh</sub>](http://bogas04.github.io) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=bogas04)  | [Ben Briggs<br /><sub>Ben Briggs</sub>](http://beneb.info) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb) | [James Monger<br /><sub>James Monger</sub>](http://github.com/Jameskmonger) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger)| [Jeroen Engels<br /><sub>Jeroen Engels</sub>](https://github.com/jfmengels) <br />  [📖](https://github.com/kentcdodds/all-contributors/commits?author=jfmengels) | [Chris Simpkins<br /><sub>Chris Simpkins</sub>](http://github.com/chrissimpkins) <br />   [📖](https://github.com/kentcdodds/all-contributors/commits?author=chrissimpkins) |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |

For 6+:

| [Kent C. Dodds<br /><sub>Kent C. Dodds</sub>](http://kentcdodds.com) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=kentcdodds) 👀 ❓ | [Divjot Singh<br /><sub>Divjot Singh</sub>](http://bogas04.github.io) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=bogas04)  | [Ben Briggs<br /><sub>Ben Briggs</sub>](http://beneb.info) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb) | [James Monger<br /><sub>James Monger</sub>](http://github.com/Jameskmonger) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger)| [Jeroen Engels<br /><sub>Jeroen Engels</sub>](https://github.com/jfmengels) <br />  [📖](https://github.com/kentcdodds/all-contributors/commits?author=jfmengels) | [Chris Simpkins<br /><sub>Chris Simpkins</sub>](http://github.com/chrissimpkins) <br />   [📖](https://github.com/kentcdodds/all-contributors/commits?author=chrissimpkins) |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| [Kent C. Dodds<br /><sub>Kent C. Dodds</sub>](http://kentcdodds.com) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=kentcdodds) 👀 ❓ | [Divjot Singh<br /><sub>Divjot Singh</sub>](http://bogas04.github.io) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=bogas04)  | [Ben Briggs<br /><sub>Ben Briggs</sub>](http://beneb.info) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb) | [James Monger<br /><sub>James Monger</sub>](http://github.com/Jameskmonger) <br /> [📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger)| [Jeroen Engels<br /><sub>Jeroen Engels</sub>](https://github.com/jfmengels) <br />  [📖](https://github.com/kentcdodds/all-contributors/commits?author=jfmengels) | [Chris Simpkins<br /><sub>Chris Simpkins</sub>](http://github.com/chrissimpkins) <br />   [📖](https://github.com/kentcdodds/all-contributors/commits?author=chrissimpkins) | 


I think we can extend @jfmengels's CLI app to gh-pages of the repo as a GUI/WYSWYG solution.

**Comment by jfmengels** [comment]
Totally dig this look. It looks much better than before. :+1: also on indicating the order of contributors.

I'm not sure the spec should really define how it should look in too much detail though, and should leave some leeway to maintainers.
- I'm not sure the avatar is always wanted. I agree that it is much, much better with it, but might become very cumbersome for big projects. Willing to know what you guys think.
- 7 columns seems very arbitrary. Let the maintainer decide IMO.
- Ditto for the image size: 100 is pretty nice IMO, but for projects with lots of contributors, reducing the size and allowing to increase the number of columns should make the spec stay in place longer (before the maintainers scrap if because it's too cumbersome I mean). Let's make 100 the recommended size I'd say?

To keep in mind: with the columns system, the use of a tool will pretty much be necessary, as it's almost not human readble/maintainable anymore.

But this is great, and much better than before in any case! :+1: 

**Note:** In `.all-contributorsrc`, `imageSize` should be set to 100 since you changed the default image size.

**Comment by jfmengels** [comment]
I like @bogas04's idea of having everything in one cell, because it makes the content easier to maintain by a "human".
Besides, they look pretty much the same and @bogas04's version takes a little less space: (ignoring the screenshot size)
image image

**Comment by kentcdodds** [comment]
Fantastic feedback. This has been updated. Let me know what else you think!

File: README.md [hunk 1]
```diff
@@ -26,13 +26,17 @@ Open source projects should include the following mandatory items in order to su
 	- Name
 	- URL link to a site where more information can be learned about the contributor.  This URL may be determined by the contributor at the project's discretion.
 	- Indication of the Contribution Category in text or icon image format using the defined Contribution Categories and/or Contribution Categories emoji images (see below).
-	- A link to the Contribution Categories key in this repository when emoji images are used as indicators of Contribution Categories.
+	- A link to the Contribution Category in text or emoji format using the defined Contribution Categories and/or [Contribution Categories emoji](#emoji-key).
+    - For every group of contributors, a new table is created.
```
Comment (kentcdodds) [inline README.md]: :+1:

**Comment by chrissimpkins** [comment]
Is it necessary to spec image size or table formatting?  This seems to be delving into too much detail with formatting issues.

**Comment by kentcdodds** [comment]
@chrissimpkins, great idea. Do you mind filing another issue and submitting a PR. I envision a `EXAMPLES.md` file :-)

Other than that, does this PR look good? I'd like to get at least 2 others to sign off on it :-)

- [ ] @chrissimpkins 
- [ ] @jfmengels 
- [ ] @ben-eb 
- [ ] @bogas04 
- [ ] @Jameskmonger

**Comment by bogas04** [comment]
LGTM

**Comment by chrissimpkins** [comment]
:+1:

**Comment by ben-eb** [comment]
:+1:

**Comment by kentcdodds** [comment]
It's not my project, it's ours :+1:

**Comment by krmaxwell** [comment]
:+1:

**Comment by Jameskmonger** [comment]
Great work all. The new styling looks great.

**Pull request merged**